lib.php

```diff
@@ -8484,12 +8484,10 @@ function hsuforum_view($forum, $course, $cm, $context) {
  */
 function hsuforum_discussion_view($modcontext, $forum, $discussion) {
     
-    if (isset($_SESSION['mod_hsuforum_page_refresh']) && $_SESSION['mod_hsuforum_page_refresh'] > 0) {
+    $pageWasRefreshed = isset($_SERVER['HTTP_CACHE_CONTROL']) && $_SERVER['HTTP_CACHE_CONTROL'] === 'max-age=0';
```

Comment:
This will not work with safari

Reply:
It's the reverted stuff that Rumano did in the hsu_forum submodule/

Reply:
Gotcha